src/job_description_feature_extraction.py

```diff
@@ -31,6 +33,8 @@ def get_one_hot_encoded_words(feature_to_extract):
 
 
 def get_feature_word_count(column_to_count):
```

Comment:
again, didn't want to delete somebody elses code, maybe it has a future purpose i dont know of yet.. still delete?